### PR TITLE
MINOR: Fix some typos

### PR DESCRIPTION
**Pull request by mojh7**
Minor typos
 - `curent` -> `current` ( MockLog.java, PlaintextAdminIntegrationTest.scala )
 - `ConcurentHashMap` -> `ConcurrentHashMap` ( Microbenchmarks.java )

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by machi1990 — APPROVED** [review]
lgtm

**Review by divijvaidya — APPROVED** [review]

**Review by Scanteianu — APPROVED** [review]

**Review by jlprat — COMMENTED** [review]
Thanks @mojh7 for your contribution.
Changes look good to me, but given that we are touching these files, can we also clean-up the unused private method `epochForEndOffset` in the `org.apache.kafka.raft.MockLog.java` file? Lines 201 - 203
Would you be up for it?

**Review by jlprat — APPROVED** [review]
Thanks @mojh7 LGTM.
I will wait until CI runs and then merge

**Comment by jlprat** [comment]
All tests seem to be flaky and unrelated to this change

**Pull request merged**
